source/site-administrators/studio/sidebar-configuration.rst

```diff
@@ -35,13 +35,20 @@ Sample
     		<params>
               <label>Dashboard</label>
     		  <path>/site-dashboard</path>
-              <icon>  	               (optional icon customization - only one state (no tree link))
```

Comment:
There are two scenarios for customizing icons in sidebar, one and two states. In this example the dashboard entry is a one state case, where there is no 'tree' opening, so we add it as it already was documented.
What you added is correct for the second scenario, where we have a tree that contains two states (closed/opened). That scenario was missing in our documentation (thanks for contributing) but it should be added in the correct place. 
What we can do is adding both scenarios (keep the current one, and add the new one below, like this

```
<modulehook>
          <name>wcm-root-folder</name>   
          <showDivider>true</showDivider> 
          <params>
             <label>Pages</label>             
             <path>/site/website</path>    
             <showRootItem>true</showRootItem>  
             <onClick>preview</onClick>
             <roles>...</roles>
             <module-icon-open>       (optional module-icon-open customization - two states (tree link))
                <class>fa-cog</class>
                <styles>
                    <color>#409a00</color>
                    <font-size>16px</font-size>
                </styles>
             </module-icon-open>
             <module-icon-closed>     (optional module-icon-closed customization - two states (tree link))
                <class>fa-cog</class>
                <styles>
                    <color>#409a00</color>
                    <font-size>16px</font-size>
                </styles>
             </module-icon-closed>
          </params>
       </modulehook>
```

That way we specify customization for both one and two states scenarios.